autogen/main/cluster.tf.tmpl

```diff
@@ -259,6 +259,7 @@ resource "google_container_cluster" "primary" {
   dynamic "master_authorized_networks_config" {
     for_each = {% if private_cluster %}var.enable_private_endpoint || {% endif %}length(var.master_authorized_networks) > 0 ? [true] : []
```

Comment:
Probably want to also create this block if `var.gcp_public_cidrs_access_enabled` is not null?